operations/serviceoverbookinglimits.md

```diff
@@ -0,0 +1,208 @@
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned and optional Cursor for the starting point of data. |
+
+### Response
+
+```javascript
+{
+  "ServiceOverbookingLimits": [
+    {
+      "Id": "e654f217-d1b5-46be-a820-e93ba568dfac",
+      "ServiceId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "FirstTimeUnitStartUtc": "2024-11-01T00:00:00Z",
+      "LastTimeUnitStartUtc": "2024-11-30T00:00:00Z",
+      "Limit": 10,
+      "CreatedUtc": "2024-01-31T10:48:06Z",
+      "UpdatedUtc": "2024-01-31T10:58:06Z",
+      "IsActive": true
+    }
+  ],
+  "Cursor": "e654f217-d1b5-46be-a820-e93ba568dfac"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ServiceOverbookingLimits` | array of [ServiceOverbookingLimit](serviceoverbookinglimits.md#serviceoverbookinglimit) | required | Service overbooking limits of the default service. |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+
+#### ServiceOverbookingLimit
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the service overbooking limit. |
+| `ServiceId` | string | required | Unique identifier of the `Service`. |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `Limit` | integer | required | Number of overbookings allowed for the `Service`. |
+| `CreatedUtc` | string | required | Timestamp in UTC timezone ISO 8601 format when the service overbooking was created. |
+| `UpdatedUtc` | string | required | Timestamp in UTC timezone ISO 8601 format when the service overbooking was updated. |
+| `IsActive` | boolean | required | Whether the service overbooking limit is still active. |
+
+## Set service overbooking limits
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Adds new service overbooking limits with the specified conditions. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+Care is needed to specify `FirstTimeUnitStartUtc` and `LastTimeUnitStartUtc` in the correct format - see [Datetimes](../guidelines/serialization.md#datetimes).
+
+It's not allowed to update past service overbooking limits outside of `EditableHistoryInterval`.
```

Comment:
oh, one more thing – I don't think we have a single parameter `EditableHistoryInterval` any more, instead we have the two: 
`AccountingEditableHistoryInterval` and `OperationalEditableHistoryInterval`, not sure which one applies here?

Reply:
Hello, to be fair, I use `EditableHistoryInterval` because this phrase is also used in `services/updateAvailability` (https://github.com/MewsSystems/gitbook-connector-api/blob/master/operations/services.md#update-service-availability). This way, the terminology is unified when the same backend logic is applied.

So my question is, do I really need to change it?

Reply:
So, `EditableHistoryInterval` is available but deprecated, in the code we pick the smaller of `AccountingEditableHistoryInterval` and `OperationalEditableHistoryInterval`.

In this case, I think the service bookings are operational data, so `OperationalEditableHistoryInterval` should be used. The terminology in this case is outdated, we should fix `updateAvailability` as well.

Reply:
Thanks for clarification, updated.